### PR TITLE
Reorder and document members in ScrollableState

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

File: packages/flutter/lib/src/widgets/scrollable.dart [hunk 1]
```diff
@@ -55,6 +55,20 @@ abstract class Scrollable extends StatefulComponent {
...
 
+  /// Whether to place zeroth child at the start of the container or
+  /// the end of it, when the scrollable has not been scrolled and has
+  /// no initial scroll offset.
+  ///
+  /// For example, if the [scrollDirection] is [Axis.vertical], then
+  /// [ViewportAnchor.start] means that the top of the zeroth item
+  /// should be aligned with the top of the scrollable, and
+  /// [ViewportAnchor.end] means the bottom of the zeroth item should
+  /// be aligned with the bottom of the scrollable.
```
Comment (abarth) [inline packages/flutter/lib/src/widgets/scrollable.dart]: This isn't accurate.  If there are 5 items, the bottom of the 5th item will be aligned with the bottom of the scrollable.

File: packages/flutter/lib/src/widgets/scrollable.dart [hunk 2]
```diff
@@ -153,10 +167,45 @@ abstract class Scrollable extends StatefulComponent {
...
-/// [ScrollableState].
+/// Widgets that subclass [Scrollable] typically use state objects
+/// that subclass [ScrollableState].
+///
+/// The main state of a ScrollableState is the [scrollOffset]. The
+/// units of the scroll offset are defined by the specific subclass.
+/// By default, the units are logical pixels.
+///
+/// The "scroll offset" is the logical description of the current
+/// scroll position. It is a double.
```
Comment (abarth) [inline packages/flutter/lib/src/widgets/scrollable.dart]: This paragraph seems slightly redundant with the previous paragraph.  Maybe merge them somehow?

**Comment by abarth** [comment]
LGTM

**Comment by Hixie** [comment]
Updated those two, PTAL?

Will check in when travis goes green.

**Comment by abarth** [comment]
LGTM

**Pull request merged**